recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -8,7 +8,7 @@ SRC_URI +=" \
           "
 
 # fix after default security flags in poky
-TOOLCHAIN_OPTIONS_append = "${SECURITY_NOPIE_CFLAGS}"
+TOOLCHAIN_OPTIONS_append_qemux86-64 = "${SECURITY_NOPIE_CFLAGS}"
```

Comment:
Please add a space before `${SECURITY_NOPIE_CFLAGS}` since its appending.

Reply:
Got it, thanks again!